lib/mtg_cli/card_display.rb

```diff
@@ -0,0 +1,82 @@
+module MtgCli
+  class CardDisplay
+    LINE_COLUMNS = 40
+
+    attr_reader :card
+
+    def initialize(card)
+      @card = card
+      @traits = card.traits.map(&:to_sym)
+    end
+
+    def to_s
+<<-CARD
+
+#{header(card.name, card.mana_cost)}
+
+#{card.type}
+
+#{wrap(card.text)}
+
+#{pt(card.power, card.toughness)}
+CARD
+    end
+
+    def verbose
```

Comment:
At the moment `#verbose` is not very DRY. You could change the signature of `#to_s` to something like `def to_s(verbose: false)` and move the additional behavior into a private method that gets called when `verbose == true`.